freezeyt/freezer.py

```diff
@@ -302,6 +304,15 @@ async def get_result(self):
                 return await get_result()
             return None
         raise MultiError(self.failed_tasks.values())
+    
+    def remove_incomplete_dir(self):
+        remove_incomplete_dir = self.config.get("remove_incomplete_dir")
+        if (
+                self.failed_tasks and
+                isinstance(self.saver, FileSaver) and
```

Comment:
But what if we add another saver?
It's much better to add a method to both, even if it will do nothing in DictSaver.